objects/src/accounts/account_id.rs

```diff
@@ -190,6 +191,23 @@ impl AccountId {
 
         Ok(())
     }
+
+    /// Creates an Account Id from a hex string. Assumes the string starts with "0x"
```

Comment:
tiny nit: we usually add periods at the end of sentences in doc comments.

Also, should we mention here the string is expected to encode the value in big-endian order?